app/ports/views.py

```diff
@@ -196,8 +196,10 @@ def portdetail_stats(request):
         'port_installations_by_month': port_installations_by_month,
         'port_installations_by_version_and_month': port_installations_by_version_and_month,
         'port_installations_by_os_stdlib_build_arch': port_installations_by_os_stdlib_build_arch,
-        'days': days,
-        'days_ago': days_ago,
+        'days': int(days),
```

Comment:
This is okay but rather than doing this conversion at multiple places in the function, I'd just do once at the top.